src/t8_geometry/t8_geometry.h

```diff
@@ -56,138 +56,60 @@ typedef enum t8_geometry_type {
-
 /**
- * Commit a geometry handler. This specifies that no geometries will
- * be added to it and makes it ready to be used.
- * \param [in,out] geom_handler An initialized but not committed geometry handler.
+ * Evaluates the geometry of a tree at a given reference point.
+ * \param [in] cmesh The cmesh
+ * \param [in] gtreeid The global id of the tree
+ * \param [in] ref_coords The reference coordinates at which to evaluate the geometry
+ * \param [in] num_coords The number of reference coordinates
+ * \param [out] out_coords The evaluated coordinates
  */
-void
-t8_geom_handler_commit (t8_geometry_handler_t *geom_handler);
-
-/* Check if a geometry handler was committed. */
-int
-t8_geom_handler_is_committed (const t8_geometry_handler_t *geom_handler);
-
-/** Return the number of registered geometries.
- * \param [in] geom_handler A geometry handler (must be initialized and may or may not be committed).
- * \return  The number of registered geometries in \a geom_handler.
- */
-size_t
-t8_geom_handler_get_num_geometries (const t8_geometry_handler_t *geom_handler);
-
-/** If a geometry handler only has one registered geometry, get a pointer to
- *  this geometry.
- * \param [in] geom_handler Must be committed and have exactly one geometry registered.
- * \return     The only registered geometry of \a geom_handler.
- * \note  Most cmeshes will have only one geometry and this function is an optimization
- *        for that special case. It is used for example in \ref t8_cmesh_get_tree_geometry.
- */
-const t8_geometry_c *
-t8_geom_handler_get_unique_geometry (const t8_geometry_handler_t *geom_handler);
-
-/**
- * Deactivate the current active tree. Can be used to reload data, after it has been moved, for example by the
- * partition-algorithm
- * 
- * \param[in,out] geom_handler The geometry handler, where the tree has to be deactivated.
- * \note \a geom_handler must be committed before calling this function.
- */
-void
-t8_geom_handler_deactivate_tree (t8_geometry_handler_t *geom_handler);
-
-/**
- * Given a geometry's name find that geometry in the geometry handler
- * and return it.
- * \param [in] geom_handler A committed geometry handler.
- * \param [in] name         The name of a geometry.
- * \return                  A pointer to the geometry or NULL if it was not found.
- */
-t8_geometry_c *
-t8_geom_handler_find_geometry (const t8_geometry_handler_t *geom_handler, const char *name);
-
 void
 t8_geometry_evaluate (t8_cmesh_t cmesh, t8_gloidx_t gtreeid, const double *ref_coords, const size_t num_coords,
                       double *out_coords);
 
+/** Evaluates the jacobian of a tree at a given reference point.
+ * \param[in] cmesh       The cmesh
+ * \param[in] gtreeid     The global id of the tree
+ * \param[in] ref_coords  The reference coordinates at which to evaluate the jacobian
+ * \param[in] num_coords  The number of reference coordinates
+ * \param[out] jacobian   The jacobian at the reference coordinates
+ */
```

Comment:
Good job on the description indent, but the param names are not indented properly ;)